var/spack/repos/builtin/packages/relion/package.py

```diff
@@ -34,29 +34,42 @@ class Relion(CMakePackage, CudaPackage):
     homepage = "http://http://www2.mrc-lmb.cam.ac.uk/relion"
     git      = "https://github.com/3dem/relion.git"
 
-    version('develop')
```

Comment:
I would keep the `develop` version, maybe be explicit where it's coming from:
```python
version('develop', branch='master')
```

Reply:
Sounds good.  Relion's `master` branch should really be considered `develop` so nothing wrong with being more explicit.